.github/workflows/tfchecks.yml

```diff
@@ -5,7 +5,12 @@ on:
   pull_request:
   workflow_dispatch:
 jobs:
-  tf-checks-example:
+  tf-checks-basic-example:
     uses: clouddrove/github-shared-workflows/.github/workflows/tf-checks.yml@1.2.2
     with:
-      working_directory: './_example/'
\ No newline at end of file
+      working_directory: './example/basic'
```

Comment:
Folder name should be `examples` not and `example`

Reply:
done